### DISCUSSION TITLE
StackBlitz - VSCode online

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#840] [Dicas]
Conhecem o [StackBlitz](https://stackblitz.com/) ?
Estou gostando bastante, um vscode online muito rápido.

**Comment by vedovelli** [comment]
Conheci hoje e fiquei muito bem impressionado!

Ficarei feliz quando trouxerem Vue.js para ele!

**Comment by vedovelli** [comment]
```We currently support creating & exporting Angular apps (based on @angular/cli) and React apps (based on create-react-app) with support for Vue & custom templates landing shortly.```

https://medium.com/@ericsimons/stackblitz-online-vs-code-ide-for-angular-react-7d09348497f4

=D

**Comment by hermesnetto** [comment]
@vedovelli o CodeSandbox tem Vue.js :)
https://codesandbox.io/

**Comment by vedovelli** [comment]
@hermesnetto Minha primeira participação aqui no Forum e já valeu a pena.

Obrigado pela dica! =D

**Comment by felipefialho** [comment]
Seguramente o melhor editor online que já vi. Impressionante.

**Comment by romulosardinha** [comment]
@hermesnetto caramba! não conhecia, testei e gostei muito. resolveu até uns incômodos que tinha no stackblitz...
